src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -242,3 +242,228 @@
     type: command
     short-summary: Update a virtual network peering.
 """
+
+helps['network'] = """
+    type: group
+    short-summary: Manages Network resources
+"""
+helps['network application-gateway'] = """
+    type: group
+    short-summary: Provides application-level routing and load balancing services 
+"""
+helps['network application-gateway address-pool'] = """
+    type: group
+    short-summary: Commands to manage app gateway address pools
```

Comment:
> Commands to  [](start = 18, length = 13)

The "commands to..." is a little repetitive. Since it's a CLI everything is a command. :)  I prefer the short, imperative version "Manage application gateway address pools" but that's my personal preference. We aren't consistent  on style and tense ("Manage foo" vs "Manages foo" vs "Commands to manage foo").